readme.md

```diff
@@ -227,7 +227,9 @@ While annotations are not specified by IIIF they are an important enabling techn
 Content Management Systems (CMS) modules that implement or leverage the IIIF APIs.
 
 - [IIIF Image Field](https://www.drupal.org/sandbox/sdellis/2421047) - Drupal 7 module that provides an easy way to add IIIF Images to content types, and configure their display. Supports Image API versions 1.0 or 2.0.
-- [UniversalViewer Omeka S Plugin](https://omeka.org/s/modules/UniversalViewer/) - Universal Viewer is a module for Omeka S that integrates UniversalViewer, a unified online player for any files, so it can display books, images, maps, audio, movies, pdf, 3D, and anything else as long as the appropriate extension is installed. Rotation, zoom, inside search, etc. may be managed too. It uses the resources of any IIIF compliant server.
+- [UniversalViewer Omeka S Module](https://omeka.org/s/modules/UniversalViewer/) - Universal Viewer is a module for Omeka S that integrates UniversalViewer, a unified online player for any files, so it can display books, images, maps, audio, movies, pdf, 3D, and anything else as long as the appropriate extension is installed. Rotation, zoom, inside search, etc. may be managed too. It uses the resources of any IIIF compliant server.
+- [IIIF Server Omeka S Module](https://omeka.org/s/modules/IiifServer/) - IIIF Server is a module for Omeka S that integrates the IIIF specifications (level 2) and a simple image server (similar to a basic IIP Image) to allow to process and share instantly images of any size and medias (pdf, audio, video, 3D . . .) in the desired formats. Any widget that supports IIIF level 2 can use this plugin. Rotation, zoom, inside search, etc. may be managed. Dynamic lists of records may be created, for example for browse pages.
```

Comment:
Again the description in the awesome list could be shorter and rely on the other end of the link for a fuller description. No need to repeat the name of the module in the description since it is the link text. So something like this:
Module for Omeka S to integrate a IIIF Image server.
Module for

Reply:
This module does not only provide an image server, it also produces IIIF Manifests. So I would say "Module for Omeka S that implements the IIIF Image and Presentation APIs" ?

Reply:
@triplingual Note that the IiifServer module will soon be splitted in two separate modules. But it is not a blocker for this PR right now...

Reply:
Shortened description per @regisrob comment.